sty/statements/statements.dtx

```diff
@@ -1368,8 +1368,10 @@ DefConstructor('\inlinedef OptionalKeyVals:omtext {}', sub {
    my $symbolnode = XML::LibXML::Element->new('symbol');
    $symbolnode->setAttribute(name=>$symb);
    $symbolnode->setAttribute("xml:id"=>makeNCName("$symb.def.sym"));
+ if ($statement_ancestor) {
    $statement_ancestor->parentNode->insertBefore($symbolnode,$statement_ancestor);
- }
+  } else {
+   Error('expected', $statement_ancestor, undef, "'$xc' missing ancestor node.");}}
```

Comment:
The human-readable bit is the symbol name ```$symb```. The ```$xc``` XPath context would look completely alien to an sTeX user.